Sources/SpeziContact/Models/Contact.swift

```diff
@@ -11,6 +11,35 @@ import SwiftUI
 
 
 /// Encodes the contact information.
+///
+/// The following example demonstrates the usage of the `Contact`.
```

Comment:
Thanks a lot for the improved documentation 🚀 
Please reference the `Contact` via double backticks, that's the DocC norm 👍